XamlControlsGallery/PageHeader.xaml

```diff
@@ -117,9 +82,9 @@
                         </AppBarButton.Icon>
                         <AppBarButton.Resources>
                             <controls:TeachingTip x:Name="ToggleThemeTeachingTip1"                   
-                            Target="{x:Bind ThemeButton}"
-                            Title="Change themes without hassle"                                        
-                            Subtitle="It's easier than ever to see control samples in both light and dark theme!">
+                                Target="{x:Bind ThemeButton}"
```

Comment:
For some reason, now the "Toggle Theme" button interlaces with the apps caption controls:

![image](https://user-images.githubusercontent.com/16122379/64385779-98fce500-d037-11e9-8548-bfe38f8d888d.png)